src/main/java/com/puppycrawl/tools/checkstyle/filters/JavaPatchFilterElement.java

```diff
@@ -53,6 +53,10 @@
     static {
         CHECK_TO_ANCESTOR_NODES_MAP.put("AvoidNestedBlocks",
                 Arrays.asList(TokenTypes.SLIST));
+        CHECK_TO_ANCESTOR_NODES_MAP.put("EmptyStatement",
+                Arrays.asList(TokenTypes.METHOD_DEF));
+        CHECK_TO_ANCESTOR_NODES_MAP.put("EqualsHashCode",
+                Arrays.asList(TokenTypes.CLASS_DEF));
```

Comment:
I agree with EqualsHashCode. Please move it to separate pull request and I can merge it to master.

Reply:
ok, separate PR https://github.com/checkstyle/patch-filters/pull/325

Reply:
thanks. please remove this EqualsHashCode from here

Reply:
ok, removed.